### ISSUE TITLE
minor issues with cloud objects mechanism

### ISSUE THREAD

**Issue by omerb01** [pywren/pywren-ibm-cloud#171]
when I try to execute map or call_async functions with "bucket", "key" and "data_steam" params (or other possible combination) using also "internal_storage" parameter - I get an error about missing params.

example:
```python
import pywren_ibm_cloud as pywren

def my_cloudobject_put(bucket, key, data_stream, internal_storage):
    cloudobject = internal_storage.put_object(data_stream.read())
    return cloudobject

pw = pywren.ibm_cf_executor()
iterdata = "..."
pw.map(my_cloudobject_put, iterdata)
results = pw.get_result()
```

**Comment by JosepSampe** [comment]
@omerb01 Checking it

**Comment by JosepSampe** [comment]
@omerb01 Fixed in #170

**Comment by omerb01** [comment]
@JosepSampe I get a different error message now:
```
AttributeError: 'InternalStorage' object has no attribute 'storage_backend'
```

**Comment by JosepSampe** [comment]
@omerb01 Yep, my mistake. Fixed in #172

**Comment by omerb01** [comment]
thanks, everything works :)
